stable/aws-for-fluent-bit/templates/service.yaml

```diff
@@ -10,12 +8,10 @@ metadata:
 spec:
   ports:
   - name: monitor-agent
-    port: {{ .Values.serviceMonitor.service.port }}
+    port: {{ .Values.k8sService.port }}
     protocol: TCP
-    targetPort: {{ .Values.serviceMonitor.service.targetPort }}
+    targetPort: {{ .Values.k8sService.targetPort }}
   selector:
```

Comment:
This is technically a backwards breaking change I think? I'm confused since they are not documented today: https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit

Reply:
Sorry, please re-explain your thinking for adding these.

Reply:
@razorsk8jz this is backwards breaking change right?